demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_float.scss

```diff
@@ -9,20 +9,10 @@
 
 //  Utility classes: float
 
-.float--right {
-    float: right !important;
-}
+// @TODO when refactoring to tailwind, we must first define breakpoints in Tailwind.
 
-.float--right-lap-up {
+.float-right-lap-up {
```

Comment:
couldn't find any occurrence. Can You tell me where its used?

Reply:
- DpPublicSurvey
- ConsolidateModal
- listentry_header
- list_users_statements